sinch/core/ports/http_transport.py

```diff
@@ -34,7 +37,10 @@ def prepare_request(self, endpoint: HTTPEndpoint) -> HttpRequest:
         url_query_params = endpoint.build_query_params()
 
         return HttpRequest(
-            headers={},
+            headers={
+                "User-Agent": f"sinch-sdk/{sdk_version} (Python/{python_version()});"
+                              f" {self.__class__.__name__}"
```

Comment:
Regarding ADR, format syntax is `sinch-sdk/{sdk_version} ({language}/{language_version}; {implementation_type}; {auxiliary_flag})` form.
Seems this format will generate : `sinch-sdk/{sdk_version} ({language}/{language_version}); {implementation_type}`
- misplaced parenthesis (after `language_version` instead end of string)
- no semi column after `implementation_type` to mark `auxiliary_flag` content (even if empty value). _ADR do not detail if optional_